entity-framework/core/providers/cosmos/limitations.md

```diff
@@ -11,8 +11,6 @@ The Cosmos provider has a number of limitations. Many of these limitations are a
 
 ## Temporary limitations
 
-- Even if there is only one entity type without inheritance mapped to a container it still has a discriminator property.
```

Comment:
Is first one also irrelevant now?

Reply:
We now provide `HasNoDiscriminator` method.